packages/core/src/util/transactionSummaryInspector.ts

```diff
@@ -201,14 +201,19 @@ export const transactionSummaryInspector: TransactionSummaryInspector =
 
     const collateral = await getCollateral(tx, inputResolver, addresses);
 
+    const withdrawals = implicit.withdrawals || 0n;
     const totalOutputValue = await totalAddressOutputsValueInspector(addresses)(tx);
     const totalInputValue = await totalAddressInputsValueInspector(addresses, inputResolver)(tx);
-    const implicitCoin = (implicit.withdrawals || 0n) + (implicit.reclaimDeposit || 0n) - (implicit.deposit || 0n);
+    const implicitCoin = withdrawals + (implicit.reclaimDeposit || 0n) - (implicit.deposit || 0n);
     const implicitAssets = await getImplicitAssets(tx);
 
     const diff = {
       assets: subtractTokenMaps([totalOutputValue.assets, totalInputValue.assets]),
-      coins: totalOutputValue.coins - totalInputValue.coins
+      // Withdrawals are a type of "own input", which must be accounted for when computing the wallet spent amount.
```

Comment:
Nice description. My first though here is that it would be better added to a type definition, so developers will see it when implementing, but it looks like we're just relying on inference here?

Reply:
good point. I'll create an explicit type and add jsdocs for it 

Reply:
I amended the commit, the only addition being the jsdoc: https://github.com/input-output-hk/cardano-js-sdk/pull/1548/commits/6166bfc973ddfbd947f99f30e69c9aa31eeec29b#diff-a9bcb5ce8c787fb688c6ba5c62595afae666260d468bd5b2ee1bd9a22e868b51R40